GANA-Core.package/CommitOutputSpike.class/instance/showHistoriansInMorph.st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+showHistoriansInMorph
+	| historyWalker sortedRevisions text |
+	historyWalker := SquotHistoryWalker new.
+	sortedRevisions := Heap sortBlock: [:a :b | (a metadata at: #timestamp) > (b metadata at: #timestamp)].
```

Comment:
I'm for more descriptive names in general, but I doubt this would help the readability of the code here - you probably get pretty fast that this sorts to entities based on their respective timestamp. If however these entities are restricted to some specific class, I would consider renaming them to reflect that in the block.